### DISCUSSION TITLE
Title set in Map Properties

### DISCUSSION THREAD

**Discussion by WereAlpaca** [AscensionGameDev/Intersect-Engine#1949] [Ideas]
My bad if this has been asked for already. I was hoping for a toggle in the Map Properties that when it's enabled it will use "Name here" instead of the actual map name. That way as a developer you can use more technical names for maps without affecting the display name in-game.

**Comment by EspectroW94** [comment]
It would be good, since sometimes we use names to identify the maps in the editor, but we want another name to be displayed. I hope they apply it.
